jhu-idc/idc-isle-dc
# Disable fits file creation and remove checks for fits from tests

This effectively removes fits file generation from our iDC setup.  
Resolves: https://github.com/jhu-idc/iDC-general/issues/445

1. This disables the Drupal Context's that is setup to generate the FITS file when media are added
2. Removes `crayfits` service from the `.env` file as well as the `docker-compose.env.yml` file. 
2. Removes any tests for FITS files. 

This does not remove the `islandora_fits` module, as that will cause a few fields to be removed that we still need.  Seemed best to leave it. 

There are a few innocuous changes to the media ingests  - these came from a `make config-export` and are not explicit changes I made. 

Note, the container is still built in `idc-isle-buildkit`, but is no longer being used. 

To test this: 

- build islandora based on this branch
- ensure that `crayfits` container is not running
- run a few ingests that ingest media (any type)
- check that there are no FITS files generate
- after the ingests, check that there are no messages in the ActiveMQ queue for FITS at `https://islandora-idc.traefik.me:8161/`.  The queue is still there, but not being used. 
- you could then run the test suite and ensure that they all pass.
